src/mobile/src/components/Post/Post.jsx

```diff
@@ -1,19 +1,42 @@
-import { StyleSheet, Text, View, useWindowDimensions } from 'react-native';
+import {
+  StyleSheet,
+  Text,
+  View,
+  useWindowDimensions,
+  Image,
+  Pressable,
+  Linking,
+} from 'react-native';
 import useSWR from 'swr';
 import RenderHtml from 'react-native-render-html';
 import { tagsStyles, baseStyles } from './styles/post';
 
 const styles = StyleSheet.create({
   post: {
-    minHeight: 300,
+    minHeight: 400,
   },
-  title: {
-    color: '#A0D1FB',
-    fontSize: 28,
-    fontWeight: 'bold',
-    letterSpacing: -1.5,
-    paddingVertical: 16,
-    textAlign: 'center',
+  postAuthor: {
+    color: 'black',
+    fontSize: 15,
+  },
+  postAvatar: {
+    height: 50,
+    width: '20%',
+  },
+  postHeader: {
+    alignItems: 'center',
+    borderBottomColor: '#cccccc',
+    borderBottomWidth: 1.5,
+    flexDirection: 'row',
+    marginBottom: 10,
+    paddingBottom: 5,
+  },
+  postInfo: {
+    width: '80%',
+  },
+  postTitle: {
+    color: '#121D59',
```

Comment:
We might want to use `#A0D1FB` just so as to be on the same page with what our web version

Reply:
![image](https://user-images.githubusercontent.com/69558780/164482403-f51b350c-3b64-42fd-b881-b491ba4d3c52.png)
 I think I used the correct color, didnt I ?

Reply:
> We might want to use `#A0D1FB` just so as to be on the same page with what our web version

I think this color is for the dark mode.